src/main/java/seedu/linkedout/logic/commands/ExitCommand.java

```diff
@@ -9,7 +9,7 @@
 
     public static final String COMMAND_WORD = "exit";
 
-    public static final String MESSAGE_EXIT_ACKNOWLEDGEMENT = "Exiting Address Book as requested ...";
+    public static final String MESSAGE_EXIT_ACKNOWLEDGEMENT = "Exiting Job Book as requested ...";
```

Comment:
Change to fit our product.
```suggestion
    public static final String MESSAGE_EXIT_ACKNOWLEDGEMENT = "Exiting LinkedOUT as requested ...";
```

Reply:
a972860c86657043b96a8bc0fbd3c3823566ffc4